server/api/events/all.get.ts

```diff
@@ -0,0 +1,8 @@
+import { PrismaClient } from "@prisma/client";
+
+export default defineEventHandler(async event => {
+  const prisma = new PrismaClient();
+  const records = await prisma.event.findMany();
```

Comment:
i think you could replace this whole thing with the following:
it might or might not need an extra async in the function header
```ts
export default defineEventHandler(event => {
return prisma.event.findMany();
});